### PR TITLE
Add rss-puller.py

### PR DESCRIPTION
**Pull request by philbudne**
rss-puller.py queries the rss-fetcher API for new stories to queue, to enable refilling the fetcher-in queue more than once a day.

### CODE REVIEW

**Comment by philbudne** [comment]
fixes https://github.com/mediacloud/story-indexer/issues/274

File: indexer/workers/fetcher/rss-puller.py [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+                    rss_fetcher_name2opt(name),
+                    rss_fetcher_name2env(name),
+                )
+                sys.exit(1)
+            assert isinstance(val, str)
+            return val
+
+        self.rss_fetcher_user = get("user")
+        self.rss_fetcher_pass = get("pass")
+        self.rss_fetcher_url = get("url")
```
Comment (kilemensi) [inline indexer/workers/fetcher/rss-puller.py:115]:
```suggestion
        for name in ["user", "pass", "url"]:
            setattr(self, rss_fetcher_name2var(name), get_rss_fetcher_value(name))
```

**Review by kilemensi — APPROVED** [review]
🚀 

---

May be `rss-api-fetcher` instead of `rss-puller`?

**Review by pgulley — APPROVED** [review]

**Comment by philbudne** [comment]
Updated to address @kilemensi comments

**Comment by philbudne** [comment]
@thepsalmist any comments?

**Review by thepsalmist — APPROVED** [review]

**Pull request merged**